webapp/locales/de.json

```diff
@@ -354,6 +354,7 @@
   },
   "post": {
     "name": "Beitrag",
+    "pinned": "Meldung",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-21T09:00:55Z" title="Monday, October 21st 2019, 11:00:55 am +02:00">Oct 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -354,6 +354,7 @@
   },
   "post": {
     "name": "Beitrag",
+    "pinned": "Gepinnt",
```


Here I would prefer a more funktional description from the users perspective like `System info` or similar …
If the user reads `Pinned` or `Gepinnt` then they may think: Oh I haven’t pinned a post …
Or?